build/lib/ios.js

```diff
@@ -27,6 +29,16 @@ class IOS {
 		this.timestamp = options.timestamp;
 	}
 
+	babelOptions() {
+		const minSupportedIosSdk = version.parseMin(require(path.join(ROOT_DIR, 'iphone', 'package.json')).vendorDependencies['ios sdk']);
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/ios.js ...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/ios.js;
  Line: 33;
-->

- :warning: build/lib/ios.js line 33 – Found non-literal argument in require (security/detect-non-literal-require)